cpp/main.cc

```diff
@@ -286,8 +286,8 @@ int main(int argc, const char **argv) {
         params.height(), params.width(), sara_size, using_mpi);
   }
   // the input measurements, if simulated
-  if (params.source() == purify::utilities::vis_source::simulation)
-    utilities::write_visibility(uv_data, out_dir + "/input.vis");
+  // if (params.source() == purify::utilities::vis_source::simulation)
+  //  utilities::write_visibility(uv_data, out_dir + "/input.vis");
```

Comment:
I think we should remove lines, not commenting them out. That's what vcs are for, right?? :)

Reply:
done!